jenkins-integration/jenkins-jobs/common/scripts/background/curl-server-metrics-loop.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -x
+
+while true ; do
```

Comment:
Yeah I thought about that, but figured it shouldn't matter at all for prod and couldn't think of a case where it would be problematic in dev, so I left it.  I guess I don't mind deleting it if you think it'd be better; just wanted to err on the side of not losing the file in case it contained something useful for debugging, but if the archiving code is working correctly then that shouldn't be an issue anymore.